macantine/tasks.py

```diff
@@ -248,3 +249,12 @@ def fill_missing_geolocation_data():
             logger.info(f"Geolocation Bot error: Unexpected exception\n{e}")
 
     logger.info(f"Geolocation Bot: Ended process for {candidate_canteens.count()} canteens")
+
+
+@app.task()
+def delete_old_historical_records():
+    if not settings.MAX_DAYS_HISTORIAL_RECORDS:
+        logger.info("Environment variable MAX_DAYS_HISTORIAL_RECORDS not set. Old history items will not be removed.")
+        return
+    logger.info(f"History items older than {settings.MAX_DAYS_HISTORIAL_RECORDS} days will be removed.")
+    call_command("clean_old_history", days=settings.MAX_DAYS_HISTORIAL_RECORDS, auto=True)
```

Comment:
est-ce que c'est interessant d'avoir un nombre de records supprimés ? Si jamais on veut debugger

Reply:
C'est logé automatiquement par django-simple-history, par exemple :
`Removed 1 historical records for <class 'data.models.canteen.Canteen'>`